wordle-web/app.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <NuxtLayout>
+    <v-app >
+      <v-container>
+        <v-btn variant ="text" @click="router.push('/')">Home</v-btn> 
```

Comment:
Instead of having the navigation buttons always present, you could have a home button in the game page and a game button on the home page? (i.e, split these into game.vue and index.vue)

Reply:
Why do that when having a nav bar like this accomplishes the same goal, and is more standard for modern website layouts?